src/main.ts

```diff
@@ -1,6 +1,9 @@
 import "#app/polyfills";
+
 // All polyfills MUST be loaded first for side effects
```

Comment:
```suggestion
import "#app/polyfills"; // All polyfills MUST be loaded first for side effects
```
So that VSCode will stop trying to add a newline here for whatever reason.